schema.graphql

```diff
@@ -5,10 +5,15 @@ type Domain @entity {
   name: String
   labelHash: String
   parent: Domain
-  subdomains: [Domain!]
+  subdomains: [Domain!] @derivedFrom(field: "parent")
   owner: Account
   creator: Account
   transactionId: Bytes
+  lockedBy: Account
+  isLocked: Boolean
+  metaData: String
```

Comment:
May be null, please make `String!`

Reply:
doing this causes a failed pending version

Reply:
Im thinking this has to have the ability to be null because a domain is created before a metadata is set

Reply:
Please disregard, i was confused by what `!` does